src/cs-utils/FrameTimings.cpp

```diff
@@ -189,7 +191,7 @@ void TimerQueryPool::end(std::string const& name) {
     if (t) {
       range.mGPUEnd = *t;
     } else {
-      std::cerr << "WARN: TimerQueryPool::End, No more Timestamps available\n";
+      spdlog::error("Failed to end timer query: No more Timestamps available!");
```

Comment:
The original was a warning. Should this too?
```suggestion
      spdlog::warn("Failed to end timer query: No more Timestamps available!");
```